### PR TITLE
Week2 day3

### PR DESCRIPTION
**Pull request by priyanka24798**

### CODE REVIEW

File: project1/application.py [hunk 1]
```diff
@@ -5,66 +5,53 @@
...
+        database.query.all()
         name= request.form.get("username")
         password = request.form.get("password")
         email = request.form.get("email-id")
-
-        print(name)
-        print(password)
-        print(email)
-        return render_template("show.html", name=name)
+        register = database(name= name,email=email,datetime = datetime.now())
```
Comment (rehana-ms) [inline project1/application.py]: you are not storing password ?

Comment (rehana-ms) [reply project1/application.py]: datetime is declared as string in your model.Are you able to insert your record.

File: project1/application.py [hunk 2]
```diff
@@ -5,66 +5,53 @@
...
+        try:
+            db.session.add(register)
+            db.session.commit()
+            print(name)
+            print(password)
+            print(email)
+            return render_template("show.html", name=name)
+        except Exception:
+            return render_template("error.html")
```
Comment (rehana-ms) [inline project1/application.py:54]: when will you redirect to error.html?can you explain the case.

File: project1/application.py [hunk 3]
```diff
@@ -43,7 +42,7 @@ def register():
         name= request.form.get("username")
         password = request.form.get("password")
         email = request.form.get("email-id")
-        register = database(name= name,email=email,datetime = datetime.now())
+        register = database(name= name,email=email,password = password, datetime = datetime.now())
```
Comment (rehana-ms) [inline project1/application.py:45]: you declared datetime as string,are you able to insert values into your table.

File: project1/import.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+db1.init_app(app1)
+db1.create_all()
+
+def main():
+    f= open("books.csv")
+    reader = csv.reader(f)
+    for isbn, tittle, author, year in reader:
+        book = Books(isbn= isbn, tittle=tittle, author=author, year=year)
+        db1.session.add(book)
+    db1.session.commit()
```
Comment (rehana-ms) [inline project1/import.py:33]: Are you able to load all books records into your table?

File: project1/application.py (hunk 1 above)
Comment (priyanka24798) [reply project1/application.py]: yes mam, I am able to insert this in database
image

File: project1/application.py (hunk 2 above)
Comment (priyanka24798) [reply project1/application.py:54]: when person tries to register again with same name and details , it goes into error page saying that email has already been registered.

File: project1/import.py (hunk 4 above)
Comment (priyanka24798) [reply project1/import.py:33]: yes mam i am able to 
load all the books ..it showed me 5001 rows after everything is done

File: project1/application.py (hunk 3 above)
Comment (priyanka24798) [reply project1/application.py:45]: image

earlier I did not add password mam, later i have added and made commit..this is the screenshot after adding password mam

Comment (rehana-ms) [reply project1/application.py:45]: ok fine.

File: project1/import.py (hunk 4 above)
Comment (rehana-ms) [reply project1/import.py:33]: :+1:

File: project1/application.py (hunk 2 above)
Comment (rehana-ms) [reply project1/application.py:54]: ok.

**Pull request merged**